### PR TITLE
Ome2020 videos

### PR DESCRIPTION
**Pull request by will-moore**
Adding more videos for OME 2020

### CODE REVIEW

**Comment by sbesson** [comment]
Looks good. Caterina's talk title should probably be updated from TBD using the first slide.

File: events/ome-community-meeting-2020/day2/index.html [hunk 1]
```diff
@@ -18,7 +18,8 @@
       vid_title3: TBD
```
Comment (pwalczysko) [inline events/ome-community-meeting-2020/day2/index.html:18]:
```suggestion
      vid_title3: "Adaptable Community Standards and Intuitive Metadata Collection Tools for Quantitative Microscopy"
```

**Comment by will-moore** [comment]
Thanks @pwalczysko.
Good to merge and deploy?

**Comment by pwalczysko** [comment]
Merging as per email.

**Pull request merged**